wire-ios-sync-engine/Source/UserSession/ZMClientRegistrationStatus.swift

```diff
@@ -158,4 +158,35 @@ extension ZMClientRegistrationStatus {
         let isAllowedToRegisterMLSCLient = DeveloperFlag.enableMLSSupport.isOn && (BackendInfo.apiVersion ?? .v0) >= .v5
         return !hasRegisteredMLSClient && isAllowedToRegisterMLSCLient
     }
+
+    var _prekeys: [IdPrekeyTuple]? {
+        prekeys.compactMap {
+            guard
+                let id = $0.keys.first?.uint16Value,
+                let prekey = $0.values.first
+            else {
+                return nil
+            }
+
+            return IdPrekeyTuple(id: id, prekey: prekey)
+        }
+    }
+
+    var _lastResortPrekey: IdPrekeyTuple? {
```

Comment:
why do we do `_`prekeys? I don't fully understand the usage. maybe renaming to prekeyTuple or something

Reply:
I can't use the tuple in Objective-c

Reply:
I'll rename